src/test/fuzz/banman.cpp

```diff
@@ -34,8 +34,9 @@ FUZZ_TARGET_INIT(banman, initialize_banman)
 {
     FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
     SetMockTime(ConsumeTime(fuzzed_data_provider));
-    const fs::path banlist_file = GetDataDir() / "fuzzed_banlist.dat";
-    fs::remove(banlist_file);
+    const fs::path banlist_file = GetDataDir() / "fuzzed_banlist";
+    fs::remove(banlist_file.string() + ".dat");
```

Comment:
same commit:

I don't understand this line. There shouldn't be any reason this instance will ever write a banlist.dat file.

Also, to make the fuzz test more useful, it should probably *create* a banlist.dat file. Extra points for creating a corrupted one. With extra bonus points for passing one in without disk access.

Reply:
Right, removed the `fs::remove` calls and replaced them with a creation of a corrupted or inaccessible banlist file.